lxd/main.go

```diff
@@ -169,10 +169,6 @@ func main() {
 	forkzfsCmd := cmdForkZFS{global: &globalCmd}
 	app.AddCommand(forkzfsCmd.Command())
 
-	// import sub-command
```

Comment:
I think we should keep the command but have it error out and redirect to `lxd recover`. This will avoid a lot of confusion for those following old forum posts or documentation.

Reply:
OK makes sense, I've fixed that now.